src/js/components/Grommet/index.d.ts

```diff
@@ -113,6 +113,22 @@ export interface GrommetProps {
         multiple?: string;
         selected?: string;
       };
+      selectMultiple?: {
+        summarizedValue?: string;
+        clearAll?: string;
+        selectAll?: string;
+        selected?: string;
```

Comment:
```suggestion
        multiple?: string;
```

Reply:
we were missing `multple` here

Reply:
a missing renaming I think 🤔 ? 

https://github.com/grommet/grommet/pull/7009/commits/f284136212498f48bd7149062b88dae47f71ac17

let me know if that's okay!

Reply:
you are correct thank you for clarifying